wordpress-mobile/WordPress-iOS
# Notifications: User Cell

### Details:
In this PR we're slightly updating the Notification(s) User Cell:

- New asset added
- Slightly tunned colors

The UI implemented here is slightly different from the original issue. For the record: placing the Follow button on the right hand side cuts way too much the area available for the Site Name / User Name.
(Plus: this UX matches the Calypso style!)

Closes #6025

### To test:
1. Launch WPiOS
2. Open the Notifications tab / Followers filter
3. Open any of the Follower Notifications

Please, verify that the Followers UI looks as expected.

Needs review: @frosty 

--

![simulator screen shot - iphone 7 - 2017-09-28 at 10 00 52](https://user-images.githubusercontent.com/1195260/30968112-97086f52-a434-11e7-8a43-87e6e3ae3ffb.png)
